.devcontainer/devcontainer.json

```diff
@@ -17,6 +17,9 @@
         "xmake.additionalConfigArguments": "--sdk=/cheriot-tools/"
       }
     }
-  }
+  },
+  "mounts" : [
+    "source=/media/phil/SONATA,target=/mnt/SONATA,type=bind"
+  ]
```

Comment:
We probably don't want to hard code *your* mount point here!

Reply:
Doh